test/onnx/pytorch_test_common.py

```diff
@@ -107,13 +107,15 @@ def wrapper(self, *args, **kwargs):
     return skip_dec
 
 
-def skipScriptTest(min_opset_version=float("inf")):
+def skipScriptTest(min_opset_version=float("inf"), reason=""):
```

Comment:
Love this

Reply:
Can `min_opset_version=float("inf")` become `-1`? Feels like `float("inf")` is a little extreme

Reply:
Clarified in #86216